pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -144,7 +144,6 @@ def __scale_batch_reset_params(trainer, model, steps_per_trial):
     trainer.weights_summary = None  # not needed before full run
     trainer.logger = DummyLogger()
     trainer.callbacks = []  # not needed before full run
-    trainer.checkpoint_callback = False  # required for saving
```

Comment:
I removed these from Tuner because ModelCheckpoint now lives in callbacks list, and this is properly backed up by Tuner already.

Reply:
cc @SkafteNicki